org.eclipse.transformer.cli/src/test/java/transformer/test/TestTransformClass.java

```diff
@@ -675,6 +677,98 @@ public void testDirectStrings() throws TransformException, IOException {
 		Assertions.assertEquals(expectedChanges, actualChanges, "Incorrect count of constant changes");
 	}
 
+    public static final String OUTER_CLASS_RESOURCE_NAME = "/xformme/Sample_OuterClass$1.class";
+
+    @Test
+    public void testOuterClass() throws TransformException, IOException {
+        consumeCapturedEvents();
+
+        final ClassActionImpl classAction = createOuterClassAction();
+
+        final String resourceName = TEST_DATA_RESOURCE_NAME + '/' + OUTER_CLASS_RESOURCE_NAME;
```

Comment:
You end up with a double slash here since `OUTER_CLASS_RESOURCE_NAME` starts with a slash.

Reply:
Thanks! That should be fixed now.